[REF] Change the linked examples to recipe repo (#158)
neurobagel/documentation: docs/infrastructure.md

Reviewer: It looks like there are a few sections in this page where files from the `neurobagel/api` repo is still referred to for setup, and need to be updated to use corresponding files from `neurobagel/recipes` instead:
- "Uploading data to the graph"
- "Adding vocabulary files to the graph database"
  - Related to an open issue https://github.com/neurobagel/recipes/issues/14
- "Updating your Neurobagel API configuration"
  > For example, the [.template-env](https://github.com/neurobagel/api/blob/main/.template-env) file in the Neurobagel API repo...

In the "Docker Compose" section, I would also update
> Run the following in the repository root (where the `docker-compose.yml` file is)

to

> Run the following in the directory containing both the `docker-compose.yml` file from the [`local_nodes` recipe](link) and the `.env` file you just created.

The participant variables modeled by Neurobagel are named using Neurobagel's own vocabulary (for more information, see this page on [controlled terms](./term_naming_standards.md)).
This vocabulary, which defines internal relationships between vocabulary terms, 
is serialized in the file [`nb_vocab.ttl`](https://github.com/neurobagel/recipes/blob/main/vocab/nb_vocab.ttl) available from the `neurobagel/recipes` repository.

> If you have cloned the api repository, you will already have downloaded the vocabulary file.

Reviewer: ```suggestion
If you have cloned this repository, you will already have downloaded the vocabulary file.
```

**The `nb_vocab.ttl` file should be added to every created Neurobagel graph database.**
This can be done using the same script we used to upload the dataset JSONLD files, [`add_data_to_graph.sh`](https://github.com/neurobagel/recipes/blob/main/scripts/add_data_to_graph.sh), which adds all `.ttl` and/or `.jsonld` files in a given directory to the specified graph.

> Run the following code (assumes you are in the `api` directory):

Author: Thanks for finding these! I thought I had caught them all.

you must explicitly specify the origin(s) for the frontend using `NB_API_ALLOWED_ORIGINS` in `.env`.

> For detailed instructions regarding the query tool see [Running cohort queries](query_tool.md).

Reviewer: ```suggestion
(Detailed instructions for using the query tool can be found in [Running cohort queries](query_tool.md).)
```

> For example, the [`template.env`](https://github.com/neurobagel/recipes/blob/main/local_node/template.env) file in the Neurobagel API repo assumes you want to allow API requests from a query tool hosted at a specific port on `localhost` (see the [Docker Compose section](#docker-compose)).

Reviewer: ````suggestion
For example, add the following line to your `.env` file to allow API requests from a query tool hosted at a specific port on `localhost` (see the [Docker Compose section](#docker-compose)).

```bash
NB_API_ALLOWED_ORIGINS="http://localhost:3000 http://127.0.0.1:3000"
```
````
